README.md

```diff
@@ -132,6 +132,8 @@ Tip: Run "bower install" instead if you are prompted password for 'https://githu
     $ cd webapp
     $ bower update
 
```

Comment:
Maybe move the note to the top so that people can see it before they try to type out `bower update`?